packages/solidity-contracts/contracts/fuelchain/FuelMessagePortal.sol

```diff
@@ -257,14 +268,14 @@ contract FuelMessagePortal is
         bytes32 sender = bytes32(uint256(uint160(msg.sender)));
         unchecked {
             //make sure data size is not too large
-            require(data.length < MAX_MESSAGE_DATA_SIZE, "message-data-too-large");
+            if (data.length > MAX_MESSAGE_DATA_SIZE) revert MessageDataTooLarge();
```

Comment:
```suggestion
            if (data.length >= MAX_MESSAGE_DATA_SIZE) revert MessageDataTooLarge();
```

?

Reply:
I think this was a typo on the original implementation. I 'll research if it is indeed a typo or not, commit this suggestion and open another PR if needed